dtracker/src/http_server/server.rs

```diff
@@ -45,7 +44,7 @@ impl Server {
     pub fn serve(&self) -> std::io::Result<()> {
         let started_msg = format!("Serving on http://0.0.0.0:{}", self.port);
         info!(started_msg);
-        println!("{}", started_msg);
+        info!("{}", started_msg);
```

Comment:
These 3 lines can be combined into 1 (the last 2 are repeated).

The `info!` macro is at the same time a formatter and a print call.

So u can just put the `format!` parameters into a single `info!` call.

Reply:
yep, you are right

Reply:
I also did a `cargo fmt --all`